jetson/filter/src/__main__.py

```diff
@@ -164,6 +184,21 @@ def _driveVelDataCallback(self, channel, msg):
         drive_vel_data = DriveVelData.decode(msg)
         self.encoder_velocities[drive_vel_data.axis] = drive_vel_data.vel_percent
```

Comment:
ig this might break soon, i think i just merged a pr that makes it vel_m_s instead of vel_percent

Reply:
yeah ill merge that into my branch before this gets merged